builtin/mainmenu/content/contentdb.lua

```diff
@@ -387,11 +412,7 @@ local function fetch_pkgs(params)
 
 		if package.aliases then
 			for _, alias in ipairs(package.aliases) do
-				-- We currently don't support name changing
-				local suffix = "/" .. package.name
-				if alias:sub(-#suffix) == suffix then
-					aliases[alias:lower()] = package.id
-				end
+				aliases[alias:lower()] = package.id
```

Comment:
Why do we need to lowercase here? I'd probably also include an `assert` to make sure we're overwriting nothing here: `assert(aliases[alias:lower()] == nil)`.

Reply:
Ids are lower cased to make them case insensitive. This applies to package.id too